### ISSUE TITLE
Missing core-js dependency

### ISSUE THREAD

**Issue by ryjo1026** [ezaquarii/openvpn-at-home#16]
Building application with ```make devel``` results in an error at ```vue-cli-service build --mode development```:
```
ERROR  Failed to compile with 1 errors

This dependency was not found:
* core-js/modules/es7.promise.finally in ./src/main.js

To install it, you can run: npm install --save core-js/modules/es7.promise.finally
```

This is solved by installing via ```npm install core-js```. 

Should this be added to the frontend package.json?

**Comment by ryjo1026** [comment]
@ezaquarii Just let me know if you don't want contributions of this type, I figured I would make them while I familiarize myself with the project.

**Comment by ezaquarii** [comment]
Yes, definitely every improvement contribution is welcome. Thank you for spotting this out and the patch.
